src/backend/app/Events/ClusterEvent.php

```diff
@@ -37,8 +35,6 @@ public function __construct(Cluster $cluster, string $type, array $data) {
 
     /**
      * Get the channels the event should broadcast on.
-     *
```

Comment:
This as well. There seem quite harmless, I understand if we don't really need them since the types are already there.

Reply:
This applies to a lot of places. If that is the intention, then I'm fine with it. 

If I'm not wrong, you can generate PHPDocs using the defined types in the function signature.